<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COF</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/pill.png</normaloff>images/pill.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="id_edit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Numero de identidad</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_edit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_login">
     <property name="text">
      <string>iniciar sesión</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_signin">
     <property name="text">
      <string>registrate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
